pwa-devdocs/src/tutorials/pwa-studio-fundamentals/add-a-static-route/index.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Add a static route
+---
+
+Magento's built in CMS system and [PageBuilder][] feature lets you create highly customized content pages for your storefront, but
+sometimes you need a page that fulfills a specific function, such as a checkout or login page.
+
+This tutorial provides steps for creating a custom, static route for these types of functional pages.
+
+By the end of this tutorial, you will know how to:
+
+-   Define a custom React component to render route content
+-   Override Venia components
+-   Add a new static route that renders the custom React component
+
+For more information on routing, see [Routing][].
+
+{: .bs-callout .bs-callout-info}
+This tutorial requires you to have a project set up using the steps provided in the [Project Setup][] tutorial.
+
+## Create a components directory
+
+If your project does not already have one, create a `components` directory under your `src` directory.
+This directory will hold your project's custom components.
+
+```sh
+mkdir -p src/components
+```
+
+## Define a custom React component
+
+Every route needs a component to render the content.
+For this tutorial, you will define a component that renders a simple message on the page.
+This component will be assigned a route in a later step.
+
+### Create component directory
+
+Create the directory to hold the code for a custom Foo component.
+
+```sh
+mkdir -p src/components/Foo
+```
+
+### Create `foo.js` module
+
+Inside the Foo component directory, create a `foo.js` file with the following content:
+
+```jsx
+/* src/components/Foo/foo.js */
+
+import React, { useEffect } from 'react';
+
+const Foo = () => {
+
+  useEffect(() => {
+    document.title = 'Foo Test Page';
```

Comment:
I know this is a simple example for a tutorial but Venia uses `react-head` to things like this. Here's an example: https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/CartPage/cartPage.js#L59.

Not sure if you really want to get into that here though, I might just drop the manipulation of `title` in this file.